census_blueprints/execute_sync.py

```diff
@@ -0,0 +1,77 @@
+import argparse
+import sys
+import requests
+import shipyard_utils as shipyard
+
+
+EXIT_CODE_INVALID_CREDENTIALS = 200
+EXIT_CODE_BAD_REQUEST = 201
+EXIT_CODE_SYNC_REFRESH_ERROR = 210
+EXIT_CODE_UNKNOWN_ERROR = 211
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--access-token', dest='access_token', required=True)
+    parser.add_argument('--sync-id', dest='sync_id', required=True)
+    args = parser.parse_args()
+    return args
+
+
+def execute_sync(sync_id, access_token):
+    """
+    Executes/starts a Census Sync
+    """
+    sync_post_api = f"https://bearer:{access_token}@app.getcensus.com/api/v1/syncs/{sync_id}/trigger"
+    api_headers = {
+        'Content-Type' : 'application/json'
+        }
+    try:
+        sync_trigger_response = requests.post(sync_post_api, headers=api_headers)
+        # check if successful, if not return error message
+        if sync_trigger_response.status_code in [200, 201]:
```

Comment:
There's more to being success than just the 200 and 201 http status codes. Is that strictly documented in their API? If not can you use something more general like >= 200 and < 300 or some logic from the requests library?